docs/technical_reference/costing/electrolyzer.rst

```diff
@@ -19,35 +24,33 @@ The following variables are constructed on the unit block (e.g., m.fs.unit.costi
 .. csv-table::
    :header: "Description", "Symbol", "Variable Name", "Default Value", "Units"
 
-   "description", ":math:`Symbol_{example}`", "variable_name", "1", ":math:`\text{dimensionless}`"
+   "Total unit capital cost", ":math:`C_{cap}`", "capital_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Fixed operating costs", ":math:`C_{op}`", "fixed_operating_cost", "10,000", ":math:`\text{USD}_{2020}\text{/yr}`"
+   "Membrane capital cost", ":math:`C_{cap,\, mem}`", "membrane_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Anode capital cost", ":math:`C_{cap,\, anode}`", "anode_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Cathode capital cost", ":math:`C_{cap,\, cathode}`", "cathode_cost", "10,000", ":math:`\text{USD}_{2020}`"
+   "Membrane replacement cost", ":math:`C_{op,\, mem}`", "membrane_replacement_cost", "10,000", ":math:`\text{USD}_{2020}\text{/yr}`"
```

Comment:
You format the units in two different ways: (1) `":math:`\text{USD}_{2020}\text{/yr}`"`, (2)  `":math:`\frac{\text{USD}_{2012}}{m^2}`"`.

I agree with @MarcusHolly, it is more readable if it is given as (1), but I would at least chose 1 way and stick with it for consistency.